package.json

```diff
@@ -90,18 +90,20 @@
     },
     "devDependencies": {
         "@types/fs-extra": "4.0.5",
+        "@types/js-yaml": "^3.12.1",
         "@types/mocha": "5.2.5",
         "@types/mustache": "0.8.32",
         "@types/node": "7.0.43",
         "@types/q": "1.5.0",
         "@types/underscore": "1.8.9",
-        "@types/js-yaml": "^3.12.1",
         "ajv": "^6.9.1",
         "assert": "1.4.1",
         "mocha": "5.2.0",
-        "tslint": "5.8.0",
         "ts-node": "7.0.1",
+        "tslint": "5.8.0",
+        "tslint-microsoft-contrib": "^6.2.0",
         "typescript": "3.3.1",
+        "typescript-tslint-plugin": "^0.5.5",
```

Comment:
for my understanding - what is difference between tslint and typescript-tslint-plugin?

Reply:
and where is this plugin used?

Reply:
It is VSCode plugin, which runs the TSLint in the IDE.